src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/ComAwareEventInfo.cs

```diff
@@ -19,6 +20,7 @@ public ComAwareEventInfo(Type type, string eventName)
             _innerEventInfo = type.GetEvent(eventName)!;
         }
 
+        [SupportedOSPlatform("windows")]
```

Comment:
It seems like the whole `ComAwareEventInfo` class should be marked as Windows-specific. In general, nothing "COM" specific will work outside of Windows.